TMattara/key-value-crud-api
# Add docker-compose.yml

**This MR depends on this MR**: https://github.com/TMattara/key-value-crud-api/pull/2 `Connect to mongoDB with mongodb npm package`

How to test it?
1. Run `git checkout add-docker` at the project folder.
2. Run [`docker-compose`](https://docs.docker.com/compose/)  `up` at the terminal opened on the project folder.
3. Check the `healthcheck`route
![image](https://user-images.githubusercontent.com/11579927/105556692-4930c380-5ce9-11eb-9df0-88929efcd452.png)
